src/internal/static/internal/css/make_lore.css

```diff
@@ -0,0 +1,173 @@
+
+#lore-page {
+    position: relative;
+    max-width: 100vw;
+}
+
+#lore-wiki-list {
+    float: left;
+    width: 20%;
+    padding: 3rem 0 40px 40px;
+    position: relative;
+    overflow-wrap: break-word;
+    hyphens: auto;
```

Comment:
Use the `word-breaking` and `hyphenated` classes (defined in `style.css`) in your HTML code instead of using these properties 🙂 (This also applies to some other selectors below.)

Reply:
This works for the title and the menu, but for some reason not for the <p> element displaying the lore content. So for that particular field, I am keeping overflow-wrap and hyphens in my local css-file.